sdks/python/apache_beam/runners/portability/portable_runner.py

```diff
@@ -90,6 +91,15 @@ def run_pipeline(self, pipeline):
           pcoll.coder_id = proto_context.coders.get_id(coder)
       proto_context.coders.populate_map(proto_pipeline.components.coders)
 
+    # Some runners won't detect the GroupByKey transform unless it has no
+    # subtransforms.  Remove all sub-transforms until BEAM-4605 is resolved.
+    for transform_id, transform_proto in list(
+        proto_pipeline.components.transforms.items()):
+      if transform_proto.spec.urn == common_urns.primitives.GROUP_BY_KEY.urn:
+        for sub_transform in transform_proto.subtransforms:
```

Comment:
Should we also log these removal for better debug-ability of the pipeline?

Reply:
I think logging this would be more noise than help. I hope to resolve BEAM-4605 soon, just probably not in time for the demo.